docs/concepts/personal-access-token.mdx

```diff
@@ -48,6 +48,40 @@ secure and rotate them if necessary.
 
 :::
 
+## Authorizing Ory Network Workspace Admin APIs
+
+Workspace API keys are used to authorize privileged operations to Ory Network Workspace admin APIs as well as the project admin
+APIs of all projects that belong to this workspace. For example, you must use a Workspace API key to create a new project in a
+workspace, or add a new B2B SSO organization to a project within that workspace.
+
+API Keys work in the context of the workspace for which they were created, as well as for all projects of that workspace. You
+create API Keys in the [Ory Console](https://console.ory.sh/).
+
+However, workspace API keys do not work for managing the data of the projects itself, such as interacting with identities,
+sessions, or OAuth2 clients. For these operations, you need to use the project API keys.
+
+### Creating API keys
+
+Follow these steps to create API keys for your workspace:
+
+1. Go to the [Ory Console](https://console.ory.sh/).
+2. Go to **Workspace settings** through the workspace menu in the top left corner.
+3. Select **API keys** in the sidebar.
```

Comment:
There is now the `<ConsoleLink />` component (and with https://github.com/ory/docs/pull/1767 it will support workspace paths as well)

```suggestion
1. Go to <ConsoleLink route="workspace.settings.apiKeys" />
```

You'll need to manually add the necessary data to `src/components/ConsoleLink/console-nav-data.ts` and `src/components/ConsoleLink/console-nav-data.ts` but you should just be able to copy & paste from console. There will most likely be autmation around this in the future.

Reply:
nvm, we don't have a `current` shortcut for workspaces yet. Then it's fine.